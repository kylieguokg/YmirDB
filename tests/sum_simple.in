set a
sum a
append a 1000 20 30 -10
get a
sum a
sum b
bye
