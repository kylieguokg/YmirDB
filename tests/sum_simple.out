> ok

> 0

> ok

> [1000 20 30 -10]

> 1040

> no such key

> bye
